### ISSUE TITLE
e2e: Generate a percentage of invalid txs with `load` command

### ISSUE THREAD

**Issue by hvanz** [cometbft/cometbft#617]
### Problem Definition

The `load` command in the e2e `runner` generates only valid transactions. A transaction is valid in the e2e app if it's of the form `".+=.+"`. This is not realistic because in production environments some transactions sent to the nodes are invalid. For instance, Gaia nodes receive between 20 to 30% of transactions that are rejected by the Hub application. 

### Proposal

Add a new field to the e2e manifest for controlling the percentage of invalid txs that `load` will generate.

**Comment by cason** [comment]
Are duplicated transactions also deemed as invalid? This is also a scenario to consider.

**Comment by lasarojc** [comment]
> Are duplicated transactions also deemed as invalid? This is also a scenario to consider.

Agree, but this would be a test to the test application, rather then CometBFT, since the application is expected to handle duplications.

**Comment by hvanz** [comment]
For testing invalid transactions, I think it doesn't matter the reason why the application considers it invalid. Duplicate transactions is an interesting scenario for testing other parts of the code. We could add a separate field to the manifest to control the percentage of duplicate transactions.

**Comment by cason** [comment]
This pretty old issue is probably still observable in the current implementation. The only to prevent this from happening, is to have the application properly rejecting duplicated transactions: https://github.com/tendermint/tendermint/issues/3865
